src/roiextractors/memmapextractors.py

```diff
@@ -2,109 +2,118 @@
 import shutil
 import tempfile
 from pathlib import Path
+from typing import Union, Optional
 
 import numpy as np
 from tqdm import tqdm
 
 from .extraction_tools import PathType, check_get_frames_args
 from .imagingextractor import ImagingExtractor
 
+PathType = Union[str, Path]
+DtypeType = Union[str, np.dtype]
+OptionalDtypeType = Optional[DtypeType]
+DtypeType = Union[str, np.dtype]
```

Comment:
This last one supposed to be Optional? Otherwise duplicate of line 14

Reply:
No, good catch!